.gitignore

```diff
@@ -0,0 +1,3 @@
+*.pyc
+*.db
+*.xml
```

Comment:
What xml files are you ignoring? We may want to consider using Git LFS to track these, unless they are easy to recreate (i.e. their creation is entirely scripted and reproducible).